actors/account/src/lib.rs

```diff
@@ -62,6 +69,38 @@ impl Actor {
         let st: State = rt.state()?;
         Ok(st.address)
     }
+
+    /// Authenticates whether the provided signature is valid for the provided message.
+    /// Errors if the authentication is invalid.
+    pub fn authenticate_message<BS, RT>(
+        rt: &mut RT,
+        params: AuthenticateMessageParams,
```

Comment:
Might need to clarify that the `params.signature` should be raw signature bytes, not a serialized `Signature` object with a type tag. I agree this is the right way to do it tho.

Reply:
Ack, good suggestion, thanks.